components/expenses/DownloadInvoicesPopOver.js

```diff
@@ -113,7 +113,7 @@ class Overlay extends React.Component {
     );
     const dateMonth = moment.utc(`${this.state.year}${month}`, 'YYYYMM');
     const dateFrom = dateMonth.toISOString();
-    const dateTo = dateMonth.endOf('month').toISOString();
+    const dateTo = dateMonth.endOf('month').subtract(1, 'month').toISOString();
```

Comment:
I don't get this line, won't we end up with something like:

```es6
dateFrom = '2020-03-01';
dateTo = '2020-02-01';

Reply:
@Betree : I don't know what I did that; I think it was a mistake and I was testing something. Anyways, I've went ahead and reverted that particular change. Let me know if everything else is okay. 👍🏼